pkg/config/config.go

```diff
@@ -1361,6 +1361,10 @@ func load(config Config, origin string, loadSecret bool) (*Warnings, error) {
 	}()
 
 	if err := config.ReadInConfig(); err != nil {
+		if IsServerless() {
+			log.Debugf("No config file")
```

Comment:
Maybe something a bit more descriptive on this. WDYT?
```suggestion
			log.Debugf("Running in serverless environment - config file is expectedly missing: %v", err)
```

Reply:
that's perfect 👍